script.js

```diff
@@ -1,12 +1,138 @@
-//You can edit ALL of the code here
+// Setup function:
 function setup() {
   const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  displayEpisodes(allEpisodes);
```

Comment:
you are displaying the episodes two times, I think you can comment out or delete this line